Files/Dialogs/BitlockerDialog.xaml

```diff
@@ -0,0 +1,35 @@
+﻿<ContentDialog
+    x:Class="Files.Dialogs.BitlockerDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local2="using:Files.Helpers"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    CornerRadius="4"
+    DefaultButton="Primary"
+    Title="Enter password to unlock the drive"
+    BorderThickness="0"
+    PrimaryButtonClick="BitlockerDialog_PrimaryButtonClick"
+    x:Name="BitlockerDriveDialog"
```

Comment:
```suggestion
    x:Name="BitlockerDialog"
```

Reply:
Apparently the name can't be the same of the class (it won't compile).
Also the others differ:
RenameDialog => NameDialog
PropertiesDialog => PropertiesDialogMarkup